scripts/flash/index.js

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env node
+
+const { promisify } = require("util");
+const Augur = require("../../src");
+const debugOptions = require("../debug-options");
+const { getPrivateKeyFromString } = require("../dp/lib/get-private-key");
+const chalk = require("chalk");
+const columnify = require("columnify");
+
+const { NetworkConfiguration } = require("augur-core");
+const getBalance = promisify(require("./get-balance"));
+const listMarkets  = promisify(require("./list-markets"));
+const designatedReport = promisify(require("./designated-report"));
+const initialReport = promisify(require("./initial-report"));
+const disputeContribute = promisify(require("./dispute-contribute"));
+const finalizeMarket = promisify(require("./finalize-market"));
+const pushTime = promisify(require("./push-time"));
+
+const commands = ["get-balance", "list-markets", "designate-report", "initial-report", "dispute", "finalize-market", "push-time"];
+const NETWORKS = ["aura", "clique", "environment", "rinkeby", "ropsten"];
+const methods = [getBalance, listMarkets, designatedReport, initialReport, disputeContribute, finalizeMarket, pushTime];
+
+async function runCommand(command, params, networks) {
+  const networkConfigurations = networks.map(NetworkConfiguration.create);
+  console.log(chalk.yellow.dim("command"), command);
+  console.log(chalk.yellow.dim("parameters"), params);
+  console.log(chalk.yellow.dim("networks"), networks);
+  for (const network of networkConfigurations) {
+    console.log(chalk.yellow("network http:"), network.http);
+    const augur = new Augur();
+    augur.rpc.setDebugOptions(debugOptions);
+    const connect = promisify(augur.connect);
+    const auth = getPrivateKeyFromString(network.privateKey);
+
+    await connect({ ethereumNode: { http: network.http }, augurNode: process.env.AUGUR_WS });
+    await methods[commands.indexOf(command)](augur, params, auth);
+
+  }
+}
+
+function parseArgs() {
```

Comment:
Suggest using a standard parameter parser like `getopt` instead of rolling your own, as the custom syntax is (to me) somewhat confusing and unclear.

Reply:
now using options parser node package